### PR TITLE
[CTP] Add digit reader workflow

### PR DESCRIPTION
**Pull request by mfasDa**
Needed for QC at simulation level

### CODE REVIEW

**Comment by mfasDa** [comment]
Needed for QC in MC: In MC QC tasks are not attached to the AOD producer workflow but run sequentially with the file readers as data source. In order for QC tasks to subscribe to CTP digits the corresponding reader needs to be provided.

@lietava Please check!

**Comment by lietava** [comment]
@mfasDa well, ok. Although it is more for @shahor02.

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "CommonUtils/ConfigurableParam.h"
+#include "Framework/ConfigParamSpec.h"
+#include "Framework/CompletionPolicyHelpers.h"
+#include "DetectorsRaw/HBFUtilsInitializer.h"
+
+using namespace o2::framework;
+
```
Comment (shahor02) [inline Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:18]: Please add here 
```

void customize(std::vector<o2::framework::CallbacksPolicy>& policies)
{
  o2::raw::HBFUtilsInitializer::addNewTimeSliceCallback(policies);
}
```

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// we need to add workflow options before including Framework/runDataProcessing
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  // option allowing to set parameters
+  std::vector<o2::framework::ConfigParamSpec> options{
+    {"inputfile", o2::framework::VariantType::String, "ctpdigits.root", {"File with CTP digits"}},
+    {"disable-mc", o2::framework::VariantType::Bool, false, {"Do not reed MC truth branch"}},
+    {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}}};
```
Comment (shahor02) [inline Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:31]: Add after this
```
o2::raw::HBFUtilsInitializer::addConfigOption(options);
```

**Review by shahor02 — CHANGES_REQUESTED** [review]

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx (hunk 1 above)
Comment (mfasDa) [reply Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:18]: Done

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx (hunk 2 above)
Comment (mfasDa) [reply Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:31]: Done

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx (hunk 1 above)
Comment (mfasDa) [reply Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:18]: Done

File: Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx (hunk 2 above)
Comment (mfasDa) [reply Detectors/CTP/workflowIO/src/digits-reader-workflow.cxx:31]: Done

**Comment by mfasDa** [comment]
CI errors unrelated

**Review by shahor02 — APPROVED** [review]

**Pull request merged**